lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -32,23 +32,30 @@
 #include "magma_logging.h"
 #include "EnumToString.h"
 #include "SessionStateEnforcer.h"
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#define DEFAULT_AMBR_UNITS 1
+#define UPF_IP "10.32.116.9"

```

Comment:
Why is this still hardcoded? UPF has N3 and N6 IP addresses. I am guessing this is the N3 IP address which should be grabbed from eth1.

Reply:
Thanks, its not used.  Anyway I ll remove.